common/maplib/map.mjs

```diff
@@ -438,6 +459,28 @@ class _SingleGameMap {
     if (tile === null) return null;
     return this.graphicAsset.getTile(layer, tile);
   }
+
+  /**
+   * Get spawn points of single map.
+   * If We have a spawn point area from (1,1) (1,5) (5,1) (5,5),
+   * we split it here to become 16 spawn points.
+   * @return {MapCoord Array} Spawn points - A list of spawn points with mapcoord type.
```

Comment:
Just {Array}, not mapCoord Array.
`@return {Array} The map coordinates inside 'spawnPoint' cell set.`